### PR TITLE
Add OKLCH and OKLab functions to reference

### PR DESCRIPTION
**Pull request by ai**
#### Summary

I added OKLab color functions to [CSS Functional Notation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions).

#### Motivation

They are part of [CSS Color Modules 4](https://www.w3.org/TR/css-color-4/).

Adding them to this list will fix Stylelint [`function-no-unknown`](https://stylelint.io/user-guide/rules/list/function-no-unknown/) since Stylelint uses this document as a source of all valid CSS function names.

#### Supporting details

[Safari TP](https://developer.apple.com/safari/technology-preview/release-notes/) should support these functions.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by teoli2003** [comment]
Is there any browser implementing them? If not, we should wait before merging this.

**Comment by ai** [comment]
> Is there any browser implementing them?

Safari TP mentions `oklch()` support in [release notes](https://developer.apple.com/safari/technology-preview/release-notes/).

**Comment by teoli2003** [comment]
Assigning myself for review (I'm reorganizing CSS Colors). Before landing this, I want to be sure the BCD information is up-to-date, and the subpages created (the mention is pretty useless without them)

**Comment by ai** [comment]
@teoli2003 I added pages for `oklab` and `oklch`

I took `lch()` and `lab()` pages and:

1. Replaced names
2. Replaced examples according to [OKLCH Converter](https://oklch.evilmartians.io/) (`oklab()` result was converted by `culori` Node.js module)
3. Replaced Safari TP release
4. Replaced link to Oklab announce article
5. Replaced `CIE Lightness` to `perceived lightness` (the creator ok Oklab used this term).
6. Replaced `C` max value to max value for sRGB and P3 colors (Rec. 2020 `C` value is `0.5` but there is no Rec. 2020 monitors yet).

I also added `oklch()` and `oklab()` to redirects.

Questions:

1. How can I add compatibility data?
2. Will `{{Specifications}}` macros work automatically?

**Comment by ai** [comment]
@teoli2003 just to clarity why I force these changes:

The reason of this PR is not to add docs but to solve problem with CSS linters. They use MDN as a source of valid functions and right now, any CSS code with `oklch()` will be marked as invalid.

File: files/en-us/web/css/color_value/oklch/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+title: oklch()
+slug: Web/CSS/color_value/oklch
+tags:
+  - CSS
+  - CSS Data Type
+  - Data Type
+  - Reference
+  - Web
+  - color
+  - oklch
```
Comment (OnkarRuikar) [inline files/en-us/web/css/color_value/oklch/index.md:11]:
```suggestion
  - oklch
  - Experimental
```

File: files/en-us/web/css/color_value/oklab/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+title: oklab()
+slug: Web/CSS/color_value/oklab
+tags:
+  - CSS
+  - CSS Data Type
+  - Data Type
+  - Reference
+  - Web
+  - color
+  - oklab
```
Comment (OnkarRuikar) [inline files/en-us/web/css/color_value/oklab/index.md:11]:
```suggestion
  - oklab
  - Experimental
```

**Comment by teoli2003** [comment]
Hi, ai! No worry, I just didn't have enough time to go back to this. I will review this PR this week!

**Comment by ai** [comment]
@teoli2003 friendly ping for the glory of OKLCH 😺

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by ai** [comment]
Thanks! Just a small question: will it deploy automatically or did I need another step to make it public?

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/oklch

**Comment by teoli2003** [comment]
It will deploy automatically, in about 24h.

**Comment by ai** [comment]
I sent PR to compat data as well https://github.com/mdn/browser-compat-data/pull/16526

**Pull request merged**